src/components/styled.ts

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
```

Comment:
I imagine this file was created to hold some styled components that may be reused across the app.

Usually it is better to keep these in the same file that is used unless you are absolutely certain that this _will_ be very reusable - [some good reading around the topic of abstractions in general](https://sandimetz.com/blog/2016/1/20/the-wrong-abstraction)